src/mesh/NodeDB.cpp

```diff
@@ -141,11 +141,6 @@ NodeDB::NodeDB()
     if (channelFileCRC != crc32Buffer(&channelFile, sizeof(channelFile)))
         saveWhat |= SEGMENT_CHANNELS;
 
-    if (!devicestate.node_remote_hardware_pins) {
```

Comment:
Is there a replacement for this?

Reply:
There is node_remote_hardware_pins_count. But then the whole if block would be a no-op, wouldn't it?

Reply:
(I meant it would be a no-op not literally, but logically: there seems to be no reason for additionally erasing node_remote_hardware_pins, considering its usage in the rest of the code.)

Reply:
That code is from @thebentern i am sure there was a reason for this.

Reply:
I believe this was to initialize value for devices that had a device state from before the addition of that property.

Reply:
@thebentern you reckon we still need this?